sample/src/main/res/layout/item_color.xml

```diff
@@ -51,7 +51,7 @@
             android:layout_height="match_parent"
             android:layout_marginEnd="@dimen/hmrc_spacing_16"
             android:gravity="start|center_vertical"
-            tools:text="Yellow (#FFFFDD00)" />
+            tools:text="Yellow (#FFBF47)" />
```

Comment:
same here, assume you need the "FF" at the front.

Reply:
Addressed in 37cfbf34f3f4f1def3882f38baa1ef74fbf5a613